fluss-flink/fluss-flink-common/src/test/java/com/alibaba/fluss/flink/source/testutils/MockDataUtils.java

```diff
@@ -0,0 +1,146 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.fluss.flink.source.testutils;
+
+import com.alibaba.fluss.flink.row.RowConverters;
+import com.alibaba.fluss.flink.source.deserializer.FlussDeserializationSchema;
+import com.alibaba.fluss.flink.source.deserializer.Order;
+import com.alibaba.fluss.metadata.Schema;
+import com.alibaba.fluss.record.LogRecord;
+import com.alibaba.fluss.row.InternalRow;
+import com.alibaba.fluss.types.DataTypes;
+import com.alibaba.fluss.types.RowType;
+
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.apache.flink.table.data.GenericRowData;
+import org.apache.flink.table.data.RowData;
+import org.apache.flink.table.types.logical.LogicalType;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Utility class for generating mock data and test infrastructure for Fluss-Flink integration tests.
+ *
+ * <p>The mock data and utilities in this class are used across various Fluss source connector tests
+ * to verify functionality like schema handling, record deserialization, and data type conversion.
+ */
+public class MockDataUtils {
+    public static final List<Order> ORDERS =
+            Arrays.asList(
+                    new Order(600, 20, 600, "addr1"),
+                    new Order(700, 22, 601, "addr2"),
+                    new Order(800, 23, 602, "addr3"),
+                    new Order(900, 24, 603, "addr4"),
+                    new Order(1000, 25, 604, "addr5"));
+
+    public static Schema getOrdersSchemaPK() {
+        return Schema.newBuilder()
+                .column("orderId", DataTypes.BIGINT())
+                .column("itemId", DataTypes.BIGINT())
+                .column("amount", DataTypes.INT())
+                .column("address", DataTypes.STRING())
+                .primaryKey("orderId")
+                .build();
+    }
+
+    public static Schema getOrdersSchemaLog() {
+        return Schema.newBuilder()
+                .column("orderId", DataTypes.BIGINT())
+                .column("itemId", DataTypes.BIGINT())
+                .column("amount", DataTypes.INT())
+                .column("address", DataTypes.STRING())
+                .build();
+    }
+
+    /**
+     * Utility method to create a readable copy of a BinaryRowData. Creates a GenericRowData with
+     * the same content as the BinaryRowData.
+     *
+     * @param binaryRow The BinaryRowData to copy
+     * @param flussRowType The RowType describing the row structure
+     * @return A GenericRowData with the same content
+     */
+    public static RowData binaryRowToGenericRow(RowData binaryRow, RowType flussRowType) {
+        // Convert Fluss RowType to Flink RowType
+        org.apache.flink.table.types.logical.RowType flinkRowType =
+                RowConverters.flussRowTypeToFlinkRowType(flussRowType);
```

Comment:
use `com.alibaba.fluss.flink.utils.FlinkConversions.toFlinkRowType` instead, so that we can remove `RowConverters`.